pcsx2/Gif.h

```diff
@@ -26,7 +26,8 @@
 #define GUNIT_WARN(...) do {} while(0)
 
 enum GIF_PATH {
-	GIF_PATH_1 = 0,
+	GIF_PATH_Unknown = -1,
```

Comment:
I would prefer to avoid to add a value to this enum. Because the value isn't checked.